packages/react/src/components/asorted/Icon/Icon.tsx

```diff
@@ -22,7 +22,7 @@ const Icon = ({
   name,
   size = 16,
   color = "currentColor",
-  weight = "Regular",
+  weight = "Medium",
```

Comment:
This is also breaking, as we might have implemented icons without the prop thinking that Regular would be the default.